src/operator/subgraph/build_subgraph.cc

```diff
@@ -180,41 +150,40 @@ bool LabelSubgraph(const Graph& g,
     std::pair<std::vector<const nnvm::Node*>,
               std::vector<const nnvm::Node*>>> non_subgraph_node_map;
   while (!node_queue.empty()) {
-    SimpleNode* cur_node = node_queue.front();
+    auto cur_node = node_queue.front();
```

Comment:
nit: Be expressive by avoiding using `auto`. `auto` is not good for readability.

Reply:
Changed.